src/js/ColorItem.js

```diff
@@ -99,7 +99,7 @@ class ColorItem {
    * @example color.replace(hsvaColorData);
    */
   replace(color, format = null) {
-    let fallback = null;
+    let fallback = '#000000';
```

Comment:
this variable should be removed here
and line 122 replaced with `this._color = QixColor();`